CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#6516](https://github.com/CocoaPods/CocoaPods/issues/6516)
 
+* Support git progress for `pod repo update` and `pod install --repo-update`
```

Comment:
nit, this line should have to empty spaces after it, see others within this file. It's for formatting.